packages/sakuli-cli/src/commands/run-command.function.ts

```diff
@@ -49,11 +49,12 @@ export const runCommand: CommandModuleProvider = (sakuli: SakuliInstance): Comma
                 }, () => Promise.resolve());
 
                await renderErrorsFromContext(sakuli.testExecutionContext);
+               process.exit(sakuli.testExecutionContext.resultState)
             } catch (e) {
                 await renderError(e);
+                process.exit(-1);
```

Comment:
I know what you mean . The intention was to not confuse this return code with one of Sakulis return codes. Becasue return code 1 of a technically correct execution means `Warning`

Reply:
Ah, okay. That contradicts common Linux/Unix Standards. 0 indicate success, everything else indicates an error. There are different [conventions concerning exit status](https://en.wikipedia.org/wiki/Exit_status). Most of the follow the mentioned rule, some like Amiga OS do not. Nevertheless, most build systems propagate a build failure if a process returns something else than 0. Therefore we should not modify the return code for `Warning` as this could possibly block builds.